verticapy/core/vdataframe/base.py

```diff
@@ -195,10 +195,12 @@ def __init__(
                 )
 
         if isinstance(input_relation, (TableSample, list, np.ndarray, dict)):
-            return self._from_object(input_relation, usecols)
```

Comment:
The __init__ function is not supposed to return anything although the self._from_object function None.